action.yml

```diff
@@ -44,13 +47,18 @@ runs:
         # packages via `Pkg.test`.
         JULIA_PKG_SERVER: ""
     - run: |
-        # The Julia command that will be executed
-        julia_cmd=( julia --color=yes --check-bounds=yes --inline=${{ inputs.inline }} --depwarn=${{ inputs.depwarn }} --project=${{ inputs.project }} -e 'using Pkg; Pkg.test(coverage=${{ inputs.coverage }})' )
-
-        # Add the prefix in front of the command if there is one
-        prefix="${{ inputs.prefix }}"
-        [[ -n $prefix ]] && julia_cmd=( "$prefix" "${julia_cmd[@]}" )
-
-        # Run the Julia command
-        "${julia_cmd[@]}"
+        JULIA_CMD="julia"
+        PREFIX="${{ inputs.prefix }}"
+        [[ -n $PREFIX ]] && JULIA_CMD="$PREFIX $JULIA_CMD"
+        echo "JULIA_CMD=$JULIA_CMD" >> $GITHUB_ENV
+      shell: bash
+    - run: |
+        ${{ env.JULIA_CMD }} --check-bounds=yes --color=yes --depwarn=${{ inputs.depwarn }} --inline=${{ inputs.inline }} --project=${{ inputs.project }} -e '
+          import Pkg
+          include(joinpath(ENV["GITHUB_ACTION_PATH"], "kwargs.jl"))
+          kwargs = Kwargs.kwargs(;
+            coverage = :(${{ inputs.coverage }}),
+            force_latest_compatible_version = :(${{ inputs.force_latest_compatible_version }}),
+          )
+          Pkg.test(; kwargs...)'
```

Comment:
Why is this split into two steps?

Reply:
Would I just change `${{ env.JULIA_CMD }}` to `"$JULIA_CMD"`

Reply:
You could leave the bash part almost as it was and then only replace `-e 'using Pkg; Pkg.test(coverage=${{ inputs.coverage }})'` with the new code.

I forgot the exact details, but there was some reason why `()` and a bash array was used. Originally I had a similar approach as this with normal double quoted variables.

Reply:
Other than that, this PR is good to be merged

Reply:
@SaschaMann Take a look now. Is this correct?

Reply:
Looks good to me! Thanks :)